services/forgotPassword.js

```diff
@@ -0,0 +1,35 @@
+const jwt = require("jsonwebtoken");
+const User = require("../db/models/User");
+const sendMailService = require("./sendemail");
+const userRepository = require("../repositories/user");
+const env = require("../configuration/env")
+
+async function setResetPasswordToken(user) {
+  const token = jwt.sign({ userId: user._id }, process.env.JWT_SECRET, { expiresIn: '1h' });
```

Comment:
move this line of code into a utils/helper.js function so you can use it wherever you need token creation